microsoft-service-bus/2.2/modules/ROOT/pages/ms-service-bus-connector-reference.adoc

```diff
@@ -0,0 +1,1693 @@
+| Target Variable a| String | Name of a variable that stores the operation's output |  |
+| Target Value a| String |  An expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  Retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |<<ServiceBusQueue>>
+|===
+
+=== For Configurations
+* <<config>>
+
+==== Throws
+* SERVICEBUS:INVALID_TOKEN
+* SERVICEBUS:RETRY_EXHAUSTED
+* SERVICEBUS:CONNECTION_EXCEPTION
+* SERVICEBUS:UNKNOWN
+* SERVICEBUS:AMQP_ERROR
+* SERVICEBUS:CONNECTIVITY
+* SERVICEBUS:REST_ERROR
+
+
+[[queueSend]]
+=== Queue Send
+`<servicebus:queue-send>`
+
+
+This operation sends a message to a queue.
+
+
+==== Parameters
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Destination Queue a| String |  The name of the queue. |  | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  Type of transactional action. |  NOT_SUPPORTED |
+| Send Correlation Id a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  Correlation strategy id for operation. |  AUTO |
+| Body a| Any |  The body of the Message |  `#[payload]` |
+| JMS Type a| String |  The JMSType identifier header of the Message |  |
+| Correlation ID a| String |  The JMSCorrelationID header of the Message |  |
+| Send Content Type a| Boolean |  Whether or not the body content type should be sent as a property |  true |
+| Content Type a| String |  The content type of the message's body |  |
+| Send Encoding a| Boolean |  Whether or not the body outboundEncoding should be sent as a Message property |  true |
+| Encoding a| String |  The encoding of the message's body |  |
+| Reply To a| <<ServiceBusJmsDestination>> |  The destination where a reply to this Message should be sent |  |
+| User Properties a| Object |  The custom user properties to set for this message |  |
+| JMSX Properties a| <<JmsxProperties>> |  The JMSX properties to set for this message |  |
+| Persistent Delivery a| Boolean |  If true, the Message is sent using the PERSISTENT JMSDeliveryMode. |  |
+| Priority a| Number |  The default JMSPriority value to use when sending the message. |  |
+| Time To Live a| Number |  Defines the default time the message is in the broker before it expires and is discarded. |  |
+| Time To Live Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit to use in the timeToLive configurations. |  |
+| Disable Message Id a| Boolean |  If true, the message is flagged to avoid generating its MessageID. |  |
+| Disable Message Timestamp a| Boolean |  If true, the message is flagged to avoid generating its sent Timestamp. |  |
```

Comment:
```suggestion
| Disable Message Id a| Boolean |  If true, flags the message to avoid generating its MessageID. |  |
| Disable Message Timestamp a| Boolean |  If true, flags the message to avoid generating its sent Timestamp. |  |
```